tracer/test/Datadog.Trace.Tests/Tagging/TagsListTests.cs

```diff
@@ -148,7 +148,9 @@ public async Task Serialization_ServiceEntrySpan()
             deserializedSpan.Tags.Should().Contain(Tags.RuntimeId, Tracer.RuntimeId);
             deserializedSpan.Tags.Should().Contain(Tags.Propagated.DecisionMaker, "-0"); // the child span is serialized first in the trace chunk
             deserializedSpan.Tags.Should().Contain(Tags.Propagated.TraceIdUpper, hexStringTraceId);
-            deserializedSpan.Tags.Count.Should().Be(customTagCount + 5);
+            deserializedSpan.Tags.Should().ContainKey(Tags.BaseService);
+            deserializedSpan.Tags[Tags.BaseService].Should().BeOneOf(TestRunners.ValidNames);
```

Comment:
I think you can probably be more specific here if you want?
```suggestion
            deserializedSpan.Tags[Tags.BaseService].Should().Be(_tracer.DefaultServiceName);
```

Reply:
Yep thanks!